src/index.ts

```diff
@@ -15,13 +15,13 @@
  * limitations under the License.
  */
 
-var debug = require('debug')('watson-middleware:index');
+const debug = require('debug')('watson-middleware:index');
 import Botkit = require('botkit');
 import AssistantV1 = require('ibm-watson/assistant/v1');
-import {Storage} from 'botbuilder';
-import {readContext, updateContext, postMessage, asCallback} from './utils';
+import { Storage } from 'botbuilder';
+import { readContext, updateContext, postMessage } from './utils';
 import deepMerge = require('deepmerge');
-import {promisify} from 'util';
+import { BotkitMessage } from 'botkit';
 
 export interface MiddlewareConfig {
```

Comment:
I just used this in my code and I think that it would be better to name it `WatsonMiddlewareConfig`.